src/content/docs/en/pages/devtools/cli/azion-cli-alpha/edge-functions/edge-functions.mdx

```diff
@@ -0,0 +1,185 @@
+---
+title: Edge function
+description: Create, list, describe, update, and delete your edge functions on the Azion platform through Azion CLI.
+meta_tags: cli, edge functions, devops
+namespace: documentation_cli_edge_functions
+menu_namespace: cliMenuAlpha
+permalink: /documentation/products/cli/edge-functions-crud/
+---
+
+## Create
+
+### Usage
+
+```bash
+$ azion create edge-function --name "<function_name>" --code "<path_to_code>" --active <true_or_false> [--args "<path_to_args_file>"] [--file "<file_path>"]
+```
+
+### Required flags
+
+#### name
+
+The `--name` flag sets the name of the edge function being created.
+
+#### code
+
+The `--code` flag sets the path to the edge function's code.
+
+#### active
+
+The `--active` flag specifies whether the edge function is active or not. Provide `true` or `false`.
+
+### Optional flags
+
+#### args
+
+The `--args` flag sets the path to the edge function's arguments JSON file.
+
+#### file
+
+The `--file` flag specifies the file path to create an edge function. You can use `-` for reading from stdin.
+
+#### help
+
+The `--help` option displays more information about the `azion create edge-function` action.
+
+---
+
+## List
+
+### Usage
+
+```bash
+$ azion list edge-function 
+```
+
+### Optional flags
+
+#### details
+
+The `--details` flag displays all relevant fields when listing.
+
+#### filter
+
+The `--filter` flag filters items by their name.
+
+#### order-by
+
+The `--order-by` flag sorts the output based on the selected field.
+
+#### page
+
+The `--page` flag returns a page of the list according to its number. Default is `1`.
```

Comment:
pra padronizar com as abaixo, não poderia ser "defines how many pages should be returned"?